src/domrobot.ts

```diff
@@ -56,23 +54,21 @@ export class ApiClient {
             params: methodParams,
         });
 
-        return request
-            .post(this.apiUrl, {
-                body: requestBody,
-                headers: {
-                    'Content-Type': 'application/json',
-                    'User-Agent': `DomRobot/${ApiClient.CLIENT_VERSION} (Node ${process.version})`,
-                },
-                jar: this.cookieJar,
-            })
-            .then(response => {
-                if (this.debugMode) {
-                    console.log(`Request (${apiMethod}): ${requestBody}`);
-                    console.log(`Response (${apiMethod}): ${response}`);
-                }
-                return response;
-            })
-            .then(response => JSON.parse(response));
+        const response = await fetch(this.apiUrl, {
+            method: "POST",
+            headers: new Headers({
+                'Content-Type': 'application/json',
+                'User-Agent': `DomRobot/${ApiClient.CLIENT_VERSION} (Node ${process.version})`,
+                Cookie: this.cookie
```

Comment:
ERROR: /home/fsoederhuyzen/work/github.com/inwx/nodejs-client/src/domrobot.ts:61:17 - The key 'Content-Type' is not sorted alphabetically

Order should be 'content-type', cookie, 'user-agent'